logdna/request_types.go

```diff
@@ -77,6 +82,17 @@ func (alert *alertRequest) CreateRequestBody(d *schema.ResourceData) diag.Diagno
 	return diags
 }
 
+func (cateroty *categoryRequest) CreateRequestBody(d *schema.ResourceData) diag.Diagnostics {
+	// This function pulls from the schema in preparation to JSON marshal
+	var diags diag.Diagnostics
+
+	// Scalars
```

Comment:
This comment isn't really needed since there are no things like lists to separate actions needed in this function.  It's obvious what it's doing.